docs/code-search/code-navigation/envvars.mdx

```diff
@@ -1,12 +1,12 @@
-# Code navigation environment variables
+# Code Navigation Environment Variables
```

Comment:
https://handbook.sourcegraph.com/company-info-and-process/communication/content_guidelines/actionable_language/#headings-and-subheadings

Headings should be in sentence case per the handbook. 

> Write in sentence case. Sentence case makes it feel less stiff and formal.

Is the handbook out-of-date? Or is there a deliberate reason to go against that guide and use title case here?